playbooks/deployHIO.yml

```diff
@@ -94,3 +94,100 @@
     become: yes
     with_items:
       - "{{ groups['master'] }}"
+
+# Install Supervisord
+
+  - name: "install supervisord"
+    become: yes
+    apt:
+        name: "{{ item }}"
+        state: present
+        update_cache: yes
+    with_items:
+            - supervisor
+
+  - name: "supervisor service log directory"
+    become: yes
+    file:
+        path: /var/log/supervisor
+        state: directory
+
+  - name: "harmonicIO service log directory"
+    become: yes
+    file:
+        path: /home/ubuntu/HarmonicIO/logs
+        state: directory
+
+  - name: "start supervisord service"
+    become: yes
+    service:
+        name: supervisor
+        state: restarted
+
+- hosts: master
+
+  tasks:
+
+  - name: "setup supervisord service configuration file for harmonicIO master"
+    template:
+        src: ../configurationTemplates/harmonic_master.conf.j2
+        dest: /etc/supervisor/conf.d/harmonic_master.conf
+        owner: root
+    become: yes
+
+    # Supervisor cannot create log files if it doesn't exist
```

Comment:
you mean it can't create them if the directory doesn't exist? do we even need a directory?